packages/react-components/src/components/VideoTile.tsx

```diff
@@ -290,32 +264,28 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
             {onRenderPlaceholder ? (
               onRenderPlaceholder(userId ?? '', placeholderOptions, DefaultPlaceholder)
             ) : (
-              <DefaultPlaceholder
-                {...placeholderOptions}
-                /* @conditional-compile-remove(one-to-n-calling) */
-                // @conditional-compile-remove(PSTN-calls)
-                strings={strings}
-              />
+              <DefaultPlaceholder {...placeholderOptions} />
             )}
           </Stack>
         )}
 
-        {showLabel && (displayName || (showMuteIndicator && isMuted)) && (
-          <Stack horizontal className={tileInfoContainerStyle}>
+        <Stack horizontal className={tileInfoContainerStyle} tokens={tileInfoContainerTokens}>
```

Comment:
If there is `showLabel && (displayName || (showMuteIndicator && isMuted)` and `participantStateString` are both false, we probably shouldn't have this empty stack